apps/meteor/client/views/admin/moderation/MessageContextFooter.tsx

```diff
@@ -1,26 +1,27 @@
-import { Button, Icon, Menu, Option, ButtonGroup } from '@rocket.chat/fuselage';
+import { Button, Menu, Option, ButtonGroup } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import React from 'react';
 import type { FC } from 'react';
 
 import useDeactivateUserAction from './hooks/useDeactivateUserAction';
 import useDeleteMessagesAction from './hooks/useDeleteMessagesAction';
-import useDismissUserAction from './hooks/useDismissUserAction';
 import useResetAvatarAction from './hooks/useResetAvatarAction';
 
 const MessageContextFooter: FC<{ userId: string }> = ({ userId }) => {
 	const t = useTranslation();
 	const { action } = useDeleteMessagesAction(userId);
 
 	return (
-		<ButtonGroup flexGrow={1}>
-			<Button flexGrow={1} onClick={action} title={t('delete-message')} aria-label={t('Moderation_Delete_all_messages')} danger>
-				<Icon name='trash' /> {t('Moderation_Delete_all_messages')}
+		<ButtonGroup width={'full'} stretch>
```

Comment:
```suggestion
		<ButtonGroup width='full' stretch>
```

Reply:
Oh, just standard format. I'll try to enforce it by ESLint, so for now it's just a recommendation, not a requirement. I should have added a `nitpick` label on my review.